openwisp_radius/base/models.py

```diff
@@ -803,7 +803,7 @@ def __str__(self):
 
 
 _get_csv_file_private_storage = PrivateFileSystemStorage(
-    location=settings.PRIVATE_STORAGE_ROOT, base_url='/radiusbatch/csv/',
+    location=settings.PRIVATE_STORAGE_ROOT, base_url=app_settings.RADIUS_API_BASEURL,
```

Comment:
can't this stay the way it is and then you can adapt the docker deployment to have:

`OPENWISP_RADIUS_API_BASEURL = 'https://radius.domain.com' `

And serve the csv files from `https://radius.domain.com/radiusbatch/csv/`

Reply:
Okay, so you want the after host origin part to be calculated as per `urls.py` ? 
That would make sense, let me look into that! :smile: